_resources/port1.0/group/bazel-1.0.tcl

```diff
@@ -256,16 +274,15 @@ pre-build {
         return -code error "build error"
     }
     if { [option bazel.run_bazel_fetch] && [option bazel.build_cmd] ne "" && [file exists ${worksrcpath}] } {
-        set cc  [compwrap::wrap_compiler cc]
-        set cxx [compwrap::wrap_compiler cxx]
         # Run fetch
         set addpath [string map {" " ":"} [option bazel.path]]
         system -W ${worksrcpath} "PATH=${addpath}:$env(PATH) [bazel::get_build_env] [option bazel.build_cmd] [option bazel.build_cmd_opts] fetch [option bazel.build_target]"
         # Patch the bazel clang wrapper script to use MacPorts compiler selection and support ccache
         foreach f [ exec find [bazel::get_bazel_build_area] -name "wrapped_clang.cc" ] {
+            ui_msg "Patching '${f}'."
```

Comment:
Do we really want this additional line of output for each file being patched? No other port or portgroup lists files as they're being patched.

Reply:
Nope.